cmd/util/cmd/epochs/transactions/deploy_epoch_relative.cdc

```diff
@@ -0,0 +1,31 @@
+import FlowClusterQC from 0xQCADDRESS
+
+transaction(name: String, 
+            code: [UInt8],
+            currentEpochCounter: UInt64,
+            numViewsInEpoch: UInt64,
+            numViewsInStakingAuction: UInt64,
+            numViewsInDKGPhase: UInt64,
+            numCollectorClusters: UInt16,
+            FLOWsupplyIncreasePercentage: UFix64,
+            randomSource: String,
+            collectorClusters: [FlowClusterQC.Cluster]) {
+
+  prepare(signer: AuthAccount) {
+
+    let currentBlock = getCurrentBlock()
+
+    signer.contracts.add(name: name, 
+            code: code,
+            currentEpochCounter: currentEpochCounter,
+            numViewsInEpoch: numViewsInEpoch - currentBlock.view,
+            numViewsInStakingAuction: numViewsInStakingAuction, 
```

Comment:
This also should be adjusted based on current block view, otherwise the first DKG phase may be shortened. Please add comments with context here as well.
```suggestion
            numViewsInStakingAuction: numViewsInStakingAuction - currentBlock.view, 
```